### PR TITLE
add openvr 1.9.15

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **openvr/1.9.15**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/openvr/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup()
+
+set(CMAKE_CXX_STANDARD 11)
```
Comment (uilianries) [inline recipes/openvr/all/CMakeLists.txt]: Why do you need it? The cmake file from author should configure it.

File: recipes/openvr/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import os
+
+
+class OpenvrConan(ConanFile):
+    name = "openvr"
+    description = "API and runtime that allows access to VR hardware from applications have specific knowledge of the hardware they are targeting."
+    topics = ("conan", "openvr", "vr", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ValveSoftware/openvr"
+    license = "BSD 3-Clause"
```
Comment (uilianries) [inline recipes/openvr/all/conanfile.py]:
```suggestion
    license = "BSD-3-Clause"
```

File: recipes/openvr/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "{}-{}".format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["BUILD_SHARED"] = self.options.shared
+        cmake.definitions["USE_LIBCXX"] = self.settings.compiler != "Visual Studio" and str(self.settings.compiler.libcxx) == "libc++"
```
Comment (uilianries) [inline recipes/openvr/all/conanfile.py]:
```suggestion
        cmake.definitions["USE_LIBCXX"] = str(self.settings.compiler.libcxx) == "libc++"
```

Visual Studio doesn't use libc++

**Comment by danimtb** [comment]
@intelligide please address the comments above. We'd like to move this PR forward if possible. Thank you!

**Pull request closed without merging**